lib/util/trip/trip_overview.dart

```diff
@@ -81,7 +85,12 @@ class TripOverview extends StatelessWidget {
               : Theme.of(context).colorScheme.onSurface.withOpacity(0.3),
         ),
       );
-      text = Text("${maxUsedSeats ?? '?'}/${trip.seats} ${S.of(context).seats}");
+      text = Text(
+        "${maxUsedSeats ?? '?'}/${trip.seats} ${S.of(context).seats}",
+        semanticsLabel: maxUsedSeats != null
+            ? S.of(context).labelXOfYseats(maxUsedSeats, trip.seats)
+            : S.of(context).labelUnknownSeats(trip.seats),
+      );
     } else {
       seatIcons = List.generate(
```

Comment:
can we add here that labelXofYSeats also gets read when tapping on the icons or would that be confusing?

Reply:
Yes I did that, that's much better